src/app/shared/session/session.query.ts

```diff
@@ -47,29 +45,8 @@ export class SessionQuery extends Query<SessionState> {
   isService$: Observable<boolean> = this.entryType$.pipe(map((entryType) => entryType === EntryType.Service));
   isNotebook$: Observable<boolean> = this.entryType$.pipe(map((entryType) => entryType === EntryType.Notebook));
   isTool$: Observable<boolean> = this.entryType$.pipe(map((entryType) => entryType === EntryType.Tool));
-  gitHubAppInstallationLink$: Observable<string> = this.entryType$.pipe(
```

Comment:
Moved all the github app installation logic into the register-github-app component